resources/views/edit.blade.php

```diff
@@ -4,7 +4,7 @@
 
     @include($form)
 
-    @if (count($errors) > 0)
+    @if (isset($errors) && count($errors) > 0)
```

Comment:
I see, it probably is your `Illuminate\View\Middleware\ShareErrorsFromSession` middleware.https://laravel.com/docs/5.2/validation#quick-displaying-the-validation-errors
Fair enough, it stays in.